cards/event-standard/template.hbs

```diff
@@ -73,13 +73,15 @@
   {{#if showExcessDetailsToggle}}
   <button class="HitchhikerCard-detailsToggle js-HitchhikerCard-detailsToggle">
     {{card.showMoreDetails.showMoreText}}
-    <span data-component="IconComponent"
-      data-opts='{ "iconName": "chevron", "classNames": "Icon--sm Icon-collapseDown" }'></span>
+    <span>
+      {{> icons/bakedInIcon iconName='chevron' classNames='Icon--sm Icon--collapseDown' }}
```

Comment:
Nit: I'm not crazy about the name `bakedInIcon`. I think "baked in" is a colloquialism that software engineers all know, but it might not be familiar to the HHs. What about `builtInIcon` or something like that.

Reply:
We discussed that in the corresponding SDK PR: https://github.com/yext/answers-search-ui/pull/1451#discussion_r662556519

Reply:
Will try to find an alternate name!

Reply:
changed to builtInIcon, I can live with it I think my head was throbbing a bit that day